tests/migration/online_migration/register_without_ltss.pm

```diff
@@ -28,6 +28,15 @@ sub run {
     set_var('SCC_ADDONS', join(',', @scc_addons));
 
     register_system_in_textmode;
+
```

Comment:
We tried other solutions without adding a new setting but we agreed that the code is more readable with `LTSS_TO_LTSS`